app/code/Magento/Review/Test/Mftf/Test/StorefrontReviewDisallowGuestsToWriteReviewsTest.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontReviewDisallowGuestsToWriteReviewsTest">
+        <annotations>
+            <features value="Review"/>
+            <stories value="Disallow guests to write reviews"/>
+            <title value="Guests are not allowed to write reviews if the option disabled in admin"/>
+            <description value="Guests should be not able write product reviews if the option disabled in configurations"/>
+            <severity value="MINOR"/>
+            <group value="catalog"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set {{DisAllowGuestsToWriteReviewsConfigData.path}} {{DisAllowGuestsToWriteReviewsConfigData.value}}" stepKey="disAllowGuestsWriteReviews"/>
+            <createData entity="SimpleSubCategory" stepKey="createCategory"/>
+            <createData entity="_defaultProduct" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+            <actionGroup ref="CliCacheCleanActionGroup" stepKey="cleanInvalidatedCaches">
+                <argument name="tags" value="config full_page"/>
```

Comment:
Hi @DmitryTsymbal, are we required to clean `full_page` cache here? Flushing before and after the test run will have a huge impact on other tests as well.

Reply:
@eduard13 I already faced with same problem when updated configs with `co:set`. And I didn't found more clear and effective solution as cleaning caches

Reply:
@eduard13 I guess the FPC cleanup is required in this particular case since further operations are performed on the cacheable pages. Even though the path for a new page might not be stored in the FPC yet, it probably still makes sense to clear FPC for the sake of stability.